rmgpy/solver/base.pyx

```diff
@@ -55,6 +55,7 @@ from rmgpy.quantity import Quantity
 from rmgpy.chemkin import getSpeciesIdentifier
 from rmgpy.reaction import Reaction
 from rmgpy.species import Species
+from rmgpy.data.rmg import getDB
```

Comment:
To reduce the complexity of our imports and the risk of adding more cyclic imports, would it make sense provide this value to the ReactionSystem directly, e.g. from the RMG class?

Reply:
Import now replaced in various files by value from the RMG class.